tests/clvm_runtime/and-2.txt

```diff
@@ -1,2 +1,2 @@
 run -i clvm_runtime '(mod (A B) (include "macros.clvm") (and (= 1 1) (= 2 4) (= 3 3)))'
-(q ())
+(q)
```

Comment:
Wow, this is nasty and unsightly and I guess something we're just going to have to get used to.